asab/web/authn/oauth/forwarder.py

```diff
@@ -149,10 +149,9 @@ def __init__(self, *args, container, service, **kwargs):
 		return method
 
 	async def _forward_get(self, request, url, client_id, client_secret):
-		headers = await self._remove_extra_headers(dict(request.headers))
 		data = dict(request.query)
 		async with aiohttp.ClientSession() as session:
-			async with session.get(url, headers=headers, params=data) as resp:
+			async with session.get(url, headers={"Authorization": request.headers.get("Authorization", "")}, params=data) as resp:
```

Comment:
The more elegant and future-proof approach is to create` headers` dict prior (just before `data = dict(request.query)` line).

```
data = dict(request.query)
headers = {
}
auth =  request.headers.get("Authorization")
if auth is not None: headers["Authorization"] = auth
    async with aiohttp.ClientSession() as session:
```